### ISSUE TITLE
react-native run-ios failed for Yoga case problem

### ISSUE THREAD

**Issue by marianotribuj** [RocketChat/Rocket.Chat.ReactNative#1321]
- Your Rocket.Chat Experimental app version: 1.20.0
- Your Rocket.Chat server version: N/A
- Device (or Simulator) you're running with: N/A

After cloning the latest version, when run react-native run-ios, I've received an error about not being able to find a folder:

My Folder:
/Rocket.Chat.ReactNative/ios/Pods/Target Support Files/yoga/yoga-prefix.pch"

Needed Folder:
/Rocket.Chat.ReactNative/ios/Pods/Target Support Files/yoga/Yoga-prefix.pch"

I've changed the case on the folder (and the included files) and it seemed to work, but maybe there's an issue somewhere.

Thanks!

**Comment by djorkaeffalexandre** [comment]
Hi! If you run `cd ios & pod install` it will be fix..
We use Pod v1.6.2, you too can downgrade your cocoapdos to this version.

**Comment by djorkaeffalexandre** [comment]
@marianotribuj works?

**Comment by marianotribuj** [comment]
Hi! Thanks for your answer. I've just tried, but it looks like either I don't have cocoapods on my path, or its not installed at all. Which seems odd, since I've seen on the logs that somehow it has been executed in the past (I have a working app with a forked version).
While writing this message I've realized  that by mistake I opened the RocketChat project, instead of the workspace (in xcode). 
Now opened the workspace, and the pods project was also opened, along with the Rocketchat, and the build was successful.
Thanks anyway!
